src/models/meeting.js

```diff
@@ -0,0 +1,26 @@
+const { Schema, model } = require('mongoose');
+
+const meetingSchema = new Schema({
+  patientName: { 
+    type: String,
+    required: true,
+    ref: 'User'
```

Comment:
why? it's just the patient's name. The user can make another person an appointment with a doctor